packages/waku/package.json

```diff
@@ -64,6 +64,7 @@
   "scripts": {
     "dev": "swc src -d dist -w --strip-leading-paths",
     "test": "vitest run",
+    "test:watch": "vitest",
```

Comment:
I personally like running the test script in watch mode locally, but happy to leave this out if you'd prefer.

Reply:
Please leave it as it's called from the monorepo root.
I hope you can use `npx vitest` instead.

Reply:
I misunderstood. Adding `test:watch` is acceptable. (I confused with changing `test`)